### PR TITLE
feat(config): add KP-SW-07 and KP-SW-08 Keemple Wall Switches

### PR DESCRIPTION
**Pull request by pawkakol1**
Templates for Keemple wall switch devices:
- KP-SW-07
- KP-SW-08
added.

### CODE REVIEW

**Comment by pawkakol1** [comment]
@zwave-js-bot fix lint

**Comment by pawkakol1** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
Why close?

**Comment by pawkakol1** [comment]
I closed it because I had problems with run checks on mobile app. I planed open pull request today again, but I see I don't need to

File: packages/config/config/devices/0x041a/kp-sw-07.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline (primary controller)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-07.json]:
```suggestion
```

File: packages/config/config/devices/0x041a/kp-sw-07.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Scene control",
+			"description": "Scene control",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-07.json]:
```suggestion
```

File: packages/config/config/devices/0x041a/kp-sw-07.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "21",
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_on_off",
+			"label": "Setting device status after power failure",
+			"description": "Define how the Switch reacts after the power supply is back on",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-07.json]:
```suggestion
```
The template exists to to have consistent labels for this functionality across devices.

File: packages/config/config/devices/0x041a/kp-sw-07.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+			"defaultValue": 0
+		},
+		{
+			"#": "24",
+			"label": "Notification when Load status change",
+			"description": "Touch Switch can send notifications to association device when state of Touch Switch’s load change.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-07.json]:
```suggestion
			"label": "Load Status Change Reports",
```

File: packages/config/config/devices/0x041a/kp-sw-07.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled only when Load condition is not changed by Z-WAVE Command",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-07.json]:
```suggestion
					"label": "Enabled (manual control only)",
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline (primary controller)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Scene control",
+			"description": "Scene control",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
			"label": "Scene Control",
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Endpoint 1",
+			"description": "Endpoint 1: Switch status report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
			"label": "Switch 1 Status",
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Endpoint 2",
+			"description": "Endpoint 2: Switch status report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
			"label": "Switch 2 Status",
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "21",
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_on_off",
+			"label": "Setting device status after power failure",
+			"description": "Define how the Switch reacts after the power supply is back on",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+			"defaultValue": 0
+		},
+		{
+			"#": "24",
+			"label": "Notification when Load status change",
+			"description": "Touch Switch can send notifications to association device when state of Touch Switch’s load change.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
			"label": "Load Status Change Reports",
```

File: packages/config/config/devices/0x041a/kp-sw-08.json [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled only when Load condition is not changed by Z-WAVE Command",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x041a/kp-sw-08.json]:
```suggestion
					"label": "Enabled (manual control only)",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
A few changes, I'll do them myself

**Review by AlCalzone — DISMISSED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**